### ISSUE TITLE
Themes don't work using Rails 4.2.3

### ISSUE THREAD

**Issue by pinzonjulian** [themilkman/jquery-tablesorter-rails#6]
The gem works great and I'm able to sort tables. However, I can't get the themes to work.
I have included the `*= require jquery-tablesorter/theme.blue` in my application.scss.
I basically need the arrows to appear in the headers since I'm using foundation for my styles.
Is there anything I might have missed?

**Comment by themilkman** [comment]
Hi there,

Just a quick shot with a few guesses
I am sure you meant the application.CSS, not .scss, right? Did you restart your Rails-Applikation and cleared the browser cache?
Did you configure Tablesorter to use the theme correctly? 
Is the theme (CSS) included in the resulting HTML-Document?

What is the result that you see? Doesn't it work at all?

Cheers
Erik

**Comment by themilkman** [comment]
Closed until further information provided.

**Comment by pinzonjulian** [comment]
Hi! I'm sorry for the delay. Here's some info so we can solve this issue:
1. I have disabled the asset pipeline because I needed to control the import order of my sass files.
2. This is why my applications file is .scss not css. Because I'm importing my files manually.
3. I have restarted my rails app and cleared the cache.
4. I have used `require jquery-tablesorter/theme.blue` in my applications.scss file. 
5. I have these three files being injected in my html:
`<link rel="stylesheet" media="all" href="/assets/jquery-tablesorter/theme.blue.self-27b3d9f862d421e77d72d1962998a2b8c5e555aed701c5e1cc3281ed562a28f2.css?body=1" data-turbolinks-track="true">`
`<script src="/assets/jquery-tablesorter/jquery.tablesorter.self-f2c988a6edbd27c58ef691372e6834099e8b33901126f5cf8dbf7632c9920018.js?body=1" data-turbolinks-track="true"></script>`
`<script src="/assets/jquery-tablesorter/jquery.tablesorter.self-f2c988a6edbd27c58ef691372e6834099e8b33901126f5cf8dbf7632c9920018.js?body=1" data-turbolinks-track="true"></script>`
`<script src="/assets/jquery-tablesorter.self-6a3cf5192354f71615ac51034b3e97c20eda99643fcaf5bbe6d41ad59bd12167.js?body=1" data-turbolinks-track="true"></script>`

I believe since I disabled the asset pipeline for this file I'm not being able to control the order in which the stylesheet are being imported to my html. Application.scss is being imported after tablesorter's stylesheet and I think this is the issue.

So, I have a couple of questions now:
1. Is there a way to manually import the tablesorter styles?
2. Is there a style that adds only the arrows to indicate sorting? I want to keep the styles I´ve already written.

Thanks!
